api/src/migrations/convert-translation-messages.js

```diff
@@ -1,10 +1,9 @@
-const _ = require('underscore'),
-  db = require('../db'),
-  properties = require('properties'),
-  DDOC_ID = '_design/medic';
+const db = require('../db'),
+      properties = require('properties'),
+      DDOC_ID = '_design/medic';
 
 const getAttachment = name => {
-  return db.medic.getAttachment(DDOC_ID, name)
+  return db.medic.getAttachment(DDOC_ID, `/translations/${name}.properties/`)
```

Comment:
The trailing slash looks wrong - is that how it's meant to be? I imagine it gets ignored when it ends up in the URL so it works either way?

Reply:
Yes, you're right. I didn't notice that before, thanks!

Reply:
Changed to remove the leading and trailing slash.